docs/switching-to-v2.md

```diff
@@ -0,0 +1,35 @@
+# Switching to V2
+## Java distribution
+The major breaking change in V2 is the new mandatory input `distribution`. This field should be specified with one of supported distributions. See [Supported distributions](../README.md#Supported-distributions) section for the list of available options.  
+Use `zulu` keyword if you would like to continue using the same distribution as in the V1.
+```yaml
+steps:
+- uses: actions/checkout@v2
+- uses: actions/setup-java@v2-preview
+  with:
+    distribution: 'zulu'
+    java-version: '11'
+    java-package: jdk # optional (jdk or jre) - defaults to jdk
+- run: java -cp java HelloWorldApp
+```
+
+**General recommendation** — configure CI with the same distribution that is used on your local dev machine.
+
+## Installing custom Java distribution from local file
+Since `distribution` input is required in V2, you should specify it as `jdkFile` to continue installing Java from local file
```

Comment:
Since the `distribution` input is required in V2, you should specify it using `jdfFile` to continue installing Java from a local file on the runner